docs/UserGuide.md

```diff
@@ -111,8 +111,83 @@ You should notice the GUI of the application pop up.
 3. For advanced users, view all feature details [here]().
 
 --------------------------------------------------------------------------------------------------------------------
+# Introducing LinkMeIn: A Quick Tour #
+_If you are new to LinkMeIn, welcome aboard! We know that navigating the world of internship applications can be 
+daunting, especially when you have multiple opportunities in sight. But worry not, because with LinkMeIn, 
+you have a reliable companion to guide you through this exciting journey._
 
-# Current Features #
+This tour aims to help you get started with LinkMeIn by introducing you to its essential features that you will be 
+using quite often. In this tour, you will be learning how to [add a company](), 
+[view its details](), 
+[edit]() and [delete]() it.
+
+Before we embark on this tour remember to download LinkMeIn and have your computer set up correctly. If you have not 
+done so, do check out the [Quick Start Guide](#quick-start) for more information.
+Now, let us follow along this tour and explore the 
+features of LinkMeIn that will transform you into a master of your internship application.
+
+## 1. Adding your first company ##
+Adding a company is the first step to tracking your internship applications. Let us start off by adding your 
+first company to LinkMeIn. 
+
+1.Type the following `add c/Tiktok r/Software Engineer s/PA n/John Tan d/10-10-2023 e/johntan@example.com 
+p/987654321 pr/high` in the command box and press Enter.
```

Comment:
I think first point should be what we want to add. 

e.g Let's say you are interested in a Software Engineer role in Apple.... (specify what the fields mean)

2. Type the following ...